Source/UnitTests/Core/MMIOTest.cpp

```diff
@@ -26,7 +26,7 @@ TEST(UniqueID, UniqueEnough)
 TEST(IsMMIOAddress, SpecialAddresses)
 {
 	// WG Pipe address, should not be handled by MMIO.
-	EXPECT_FALSE(MMIO::IsMMIOAddress(0xCC008000));
+	EXPECT_FALSE(MMIO::IsMMIOAddress(0x0C008000));
 
 	// Memory zone used by games using the "MMU Speedhack".
```

Comment:
Not that it's from this patch, but isn't this the L1 cache area, not the vmem speedhack